test/globalInitialization.ts

```diff
@@ -13,7 +13,7 @@ before(() => {
   if (window.PHANTOMJS) {
     window.Pixel_CloseTo_Requirement = 2;
   } else if (isFirefox) {
-    window.Pixel_CloseTo_Requirement = 1;
+    window.Pixel_CloseTo_Requirement = 2;
```

Comment:
We probably don't since that should be a ratio (be dependent on the size), while we use it as a constant